meetings/membership-criteria.md

```diff
@@ -0,0 +1,64 @@
+# Membership criteria
+
+When you apply for membership, we review your application based on the [membership participation types](membership-participation-types.md) defined. We ask for some basic information about you. 
+
+## Basic information
+
+First, we validate the personal information:
+
+- First and Last name are reviewed to eliminate potential 'bots'.
+- A valid email address: We only validate that it is a valid email address format. Later, when an invitation is sent, that verifies that it's a real address.
+- A valid GitHub ID: We validate the profile appears to be human. There should be some activity. The user name must match any contribution links provided. GitHub activity mentioned in the contribution statement should match this ID.
+
+## Open Source Maintainer, Open Source Contributor
+
+See [Open source maintainer](membership-participation-types.md#the-open-source-maintainer) and the [Open source contributor](membership-participation-types.md#the-open-source-contributor).
+
+We look at the contribution links. All activity must conform to the .NET Foundation [Code of Conduct (CoC)](https://dotnetfoundation.org/about/code-of-conduct). Activities examined include:
+
+- PRs created. We evaluate the participation in the community. Does the PR match the project's guidelines? Does the author engage constructively with reviewers?
+- PR review, and helping new contributors. Are comments in line with the spirit of the CoC? Does the reviewer provide necessary assistance?
+- Issues created. These should be well-defined issues. We don't evaluate issue quality based on the project maintainer's decision to accept or reject the idea. We do evaluate issues based on the clarity and completeness.
+- Discussion forums. Does the person participate in discussions in a constructive manner?
+
+A person need not participate in all these areas, but should be active in some of them.
+
+## Event organizer
+
+See [Event organizer particpation type](membership-participation-types.md#the-event-organizer).
+
+We look at the event or user group activities. Examples include:
+
+- Events listed have a CoC that reflects the .NET Foundation's community guidelines.
+- Consistent track record holding events (virtual or in person).
+- Meetup is listed with the .NET Foundation
```

Comment:
@ovebastiansen pointed out the meetup being listed with the .NET Foundation is restrictive